dataeng/resources/model-transfers.sh

```diff
@@ -28,7 +28,11 @@ fi
 
 ARGS="{mart: ${MART_NAME} }"
 
-dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
+DBT_TARGET="prod"
```

Comment:
Do we need to hardcode this to prod? 
DBT_TARGET is a jenkins job level parameter.
https://github.com/edx/jenkins-job-dsl/blob/a2d7d190f1576b6f8a6e76953957cc0f8fc8909a/dataeng/jobs/analytics/ModelTransfers.groovy#L22

Reply:
nah, though its prod everywhere in analytics secure but i can leave it